modules/uipath/src/backend/index.ts

```diff
@@ -0,0 +1,43 @@
+import * as sdk from 'botpress/sdk'
+
+const onServerStarted = async (bp: typeof sdk) => {
+  const router = bp.http.createRouterForBot('uipath', {
+    checkAuthentication: false
+  })
+
+  router.post('/message', async (req, res) => {
+    try {
+      const body = req.body
+
+      await bp.events.replyToEvent(
+        {
+          channel: body.channel,
+          target: body.target,
+          botId: body.botId,
+          threadId: body.threadId
+        },
+        [body.message]
+      )
```

Comment:
You should add some sort of authentication here, otherwise anyone can call this route with any type of data (maybe UiPath could send an API key as header field?) 

You can use deconstructor to avoid repeating body `const { channel, target, botId, threadId } = req.body`

Also, what is the payload "message" ? Is it only a string ? I think there should be some validation on the data received